metadata/demographic/census_places_2020.md

```diff
@@ -0,0 +1,77 @@
+# Title
+
+Census 2020 Places
+
+## ID
+
+b5f022ca-96c8-4f16-bc47-8bbf35c67db9
+
+## Brief Summary
+
+Incorporated towns and Census Designated Places (CDPs) in Utah from the 2020 Census.
+
+## Summary
+
+Census Places consist of both legally incorporated cities and Census Designated Places (CDPs). This dataset contains boundary information only and does not include demographic data.
+
+## Description
+
+### What is the dataset?
+
+This dataset contains polygon boundaries for cities, towns, metro townships, and census-designated places (CDPs) in Utah. The [US Census Bureau](https://www.census.gov/programs-surveys/bas/information/cdp.html) defines CDPs as "statistical geography representing closely settled, unincorporated communities that are locally recognized and identified by name." Both these unincorporated communities and legally incorporated areas of development are represented in this dataset.
+
+### What is the purpose of the dataset?
+
+The US Census Bureau uses both legally incorporated (cities, towns, etc) and unincorporated places (CDPs) for the once-a-decade count of population and housing. While this dataset does not contain demographic information, it can be joined to other layers to visualize population data.
+
+Geography has always been a fundamental aspect of the census. You can read more about how geography is used in the census on the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).
+
+<!--- Are the data also involved in the redistricting process? -->
+
+### What does the dataset represent?
+
+Incorporated cities and CDPs are represented as polygons and can be distinguished by the CLASSFP20 field. Each polygon represents the geographic extent of a given place and includes the place name and type.
+
+<!--- I'm planning on including more information about the CLASSFP20 field once I hear back from Greg, but are there other fields that would be helpful to include here? -->
+
+### How was the dataset created?
+
+Utah's 2020 census geographic boundaries became available in February 2021 and the demographic redistricting legacy files (the 2020 Census Redistricting P.L. 94-171 Summary Files) became available on August 12, 2021. The Census Bureau released a user-friendly version of the 2020 Census Redistricting P.L. 94-171 Summary Files on September 30, 2021. UGRC, in collaboration with the Analytics Group at [WFRC](https://wfrc.org/) used that August 12 release to create ready-to-use GIS data in several formats. You can learn more about these offerings on [our website](https://gis.utah.gov/blog/2021-08-31-census-2020-redistricting-data/).
```

Comment:
**question**: i would be interested to read how ugrc and wfrc used the census data to create this. I assume some decisions were made to pick and choose parts of the census data rather than simply mirroring it. I assume it is filtered etc. Understanding those choices could help a consumer choose between this data and going to the source.

Reply:
I agree that would be helpful. @gregbunce do you have more insight on this?

Reply:
this [post](https://gis.utah.gov/blog/2021-08-31-census-2020-redistricting-data/) explains it a bit. it's also linked on the data page for this layer.